frontend/src/studio-ui/dev-preview.ts

```diff
@@ -0,0 +1,17 @@
+import { renderEditor } from './index';
+
+const mockRuntime: XBlockRuntime = {
+  handlerUrl: (e, t) => `${e}-${t}`,
+  // eslint-disable-next-line no-console
+  notify: (action, data) => console.log(`${action} ${JSON.stringify(data)}`),
+};
+
+renderEditor(mockRuntime, document.querySelector('#root'), {
+  panels: [
+    {
+      title: 'test',
+      contents: '<b>Test</b>',
+    },
+  ],
+  styling: {},
+});
```

Comment:
This is used for the developer preview when developing the component.

Reply:
Worth adding as a comment right in the code.